os/pm/Makefile

```diff
@@ -23,6 +23,10 @@ ASRCS =
 CSRCS += pm_activity.c pm_changestate.c pm_checkstate.c pm_initialize.c
 CSRCS += pm_register.c pm_unregister.c pm_update.c pm_procfs.c
 
+ifeq ($(CONFIG_PM),y)
```

Comment:
When CONFIG_PM is disabled, this Makefile will not be called. When this Makefile is called, CONFIG_PM is always enabled. Let's remove this conditional.